omnidocs/tasks/table_extraction/extractors/surya_table.py

```
@@ -0,0 +1,380 @@
+from pathlib import Path
+from typing import Union, List, Dict, Any, Optional, Tuple
+import torch
```

Ruff (0.12.2)

`torch` imported but unused

Remove unused import: `torch`

(F401)